Source/Core/Common/StringUtil.cpp

```diff
@@ -328,6 +328,25 @@ void SplitString(const std::string& str, const char delim, std::vector<std::stri
   output.pop_back();
 }
 
+std::string JoinString(const std::vector<std::string>& strings, char delim)
```

Comment:
Joining strings can be done like so, which also allows for joining via delimeters consisting of more than one char:
```cpp
std::string JoinStrings(const std::vector<std::string>& strings, const std::string& delimeter)
{
    std::stringstream res;
    std::copy(strings.begin(), strings.end(),
              std::ostream_iterator<string>(res, delimeter.c_str()));

    // Drop the trailing delimeter.
    std::string joined = res.str();
    return joined.substr(0, joined.length() - delimeter.length());
}
```